listener/wal_transaction.go

```diff
@@ -279,7 +279,7 @@ func (w *WalTransaction) CreateEventsWithFilter(ctx context.Context) []*publishe
 		event.Action = item.Kind.string()
 		event.Data = data
 		event.DataOld = dataOld
-		event.EventTime = *w.CommitTime
+		event.EventTime = time.Now()
```

Comment:
We're not actually using this in any meaningful way, so I figure we just `time.Now()` and call it a day